src/shared/api/auth/sign-up.ts

```diff
@@ -0,0 +1,22 @@
+import { http } from 'shared/api/axios';
+
+type TSignUp = {
+  first_name: string;
+  second_name: string;
+  login: string;
+  email: string;
+  password: string;
+  phone: string;
+};
+
+export const signUp = ({ first_name, second_name, login, email, password, phone }: TSignUp) =>
+  http.getInstance().post('/auth/signup', {
+    data: {
+      first_name,
+      second_name,
+      login,
+      email,
+      password,
+      phone,
+    },
```

Comment:
```suggestion
export const signUp = (data: TSignUp) =>
  http.getInstance().post('/auth/signup', {
    data,
```

Кажется тут можно вот так

Reply:
да, согласен))